### DISCUSSION TITLE
Which http version does Kyuubi rest uses? Does it support http2 or http3?

### DISCUSSION THREAD

**Discussion by HaoYang670** [apache/kyuubi#6138] [Q&A] [answered]
As the title.

**Comment by pan3793** [comment, marked as answer]
HTTP/1.

Any specific reasons for the expectation of HTTP/2 or HTTP/3?

**Reply by HaoYang670** [reply]
Thanks!
We get some performance bottleneck when the throughput is high. I am wondering if using http2 could solve some part of it.
